textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/nlpextrator/NlpExtractor.java

```diff
@@ -73,12 +111,13 @@ public void open() throws Exception {
 
 
     /**
-     * @about Return all named entities that are recognized in a document.
-     * Return format is a Tuple that contains only one field which is
-     * a list of spans of the result.
-     * @overview First get a tuple from the source operator then process it
-     * using the Stanford NLP package. for all recognized words, compute their
-     * spans and return all as a list.
+     * @about Return the extracted data as a list of spans
+     * and appends to the original tuple as a new field.
```

Comment:
Global comment: sometimes you use "appends" (with the "s") but sometimes you use "append".  Make the comments use the consistent style (i.e., with the "s").